src/api/api.go

```diff
@@ -232,7 +233,10 @@ func (a *Api) About(c *gin.Context) {
 // @Param data body RegisterNumberRequest false "Additional Settings"
 // @Router /v1/register/{number} [post]
 func (a *Api) RegisterNumber(c *gin.Context) {
-	number := c.Param("number")
+	number, err := url.PathUnescape(c.Param("number"))
+	if err != nil {
+		c.JSON(400, Error{Msg: "Couldn't process request - malformed number"})
+	}
```

Comment:
Thanks for the PR!

In general it looks good, the only thing that is missing is a `return` in case a malformed number is provided.

Reply:
Thanks, fixed.